backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/IteratorApiImpl.scala

```diff
@@ -184,8 +179,6 @@ class IteratorApiImpl extends IteratorApi with Logging {
         rootNode.toProtobuf.toByteArray,
         columnarNativeIterator)
 
-    pipelineTime += TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - beforeBuild)
```

Comment:
thanky you @Yohahaha . Shall we add one more metrics for the durationMs of preparing the native input iterator ? So the previous pipeline time can be broken into two parts, one is for preparing and the other is for executing.

Reply:
ok, I can open an issue to track it.

Reply:
https://github.com/oap-project/gluten/issues/3830